packages/larva-js/src/interface/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * JS class to create/handle tooltips
```

Comment:
Add details here about the required markup and any required CSS for using this feature - e.g. are there any specific patterns that it works well with, or how the classes should be added to the DOM. Thanks!!

Reply:
Hey @laras126, Thanks I have updated the PR to add the documentation for the script let me know if any change required.